### PR TITLE
Attempt to gather more info from correct namespaces during e2e.

### PR DESCRIPTION
**Pull request by dgoodwin**

### CODE REVIEW

File: hack/e2e-test.sh [hunk 1]
```diff
@@ -205,10 +217,15 @@ then
 fi
 
 # Capture install logs
-if INSTALL_JOB_NAME=$(oc get job -l "hive.openshift.io/cluster-deployment-name=${CLUSTER_NAME},hive.openshift.io/install=true" -o name) && [ "${INSTALL_JOB_NAME}" ]
+if IMAGESET_JOB_NAME=$(oc get job -l "hive.openshift.io/cluster-deployment-name=${CLUSTER_NAME},hive.openshift.io/imageset=true" -o name -n ${CLUSTER_NAMESPACE}) && [ "${IMAGESET_JOB_NAME}" ]
+then
+	oc logs -c hive -n ${CLUSTER_NAMESPACE} ${IMAGESET_JOB_NAME} &> "${ARTIFACT_DIR}/hive_imageset_job.log" || true
+	oc get job ${IMAGESET_JOB_NAME} -n ${CLUSTER_NAMESPACE} -o yaml &> "${ARTIFACT_DIR}/hive_imageset_job.yaml" || true
```
Comment (staebler) [inline hack/e2e-test.sh:223]: The `job` part is redundant with the job in `IMAGESET_JOB_NAME`, and may not even be valid.

File: hack/e2e-test.sh [hunk 2]
```diff
@@ -205,10 +217,15 @@ then
...
+fi
+if INSTALL_JOB_NAME=$(oc get job -l "hive.openshift.io/cluster-deployment-name=${CLUSTER_NAME},hive.openshift.io/install=true" -o name -n ${CLUSTER_NAMESPACE}) && [ "${INSTALL_JOB_NAME}" ]
 then
-	oc logs -c hive ${INSTALL_JOB_NAME} &> "${ARTIFACT_DIR}/hive_install_job.log" || true
-	oc get ${INSTALL_JOB_NAME} -o yaml &> "${ARTIFACT_DIR}/hive_install_job.yaml" || true
+	oc logs -c hive -n ${CLUSTER_NAMESPACE} ${INSTALL_JOB_NAME} &> "${ARTIFACT_DIR}/hive_install_job.log" || true
+	oc get job ${INSTALL_JOB_NAME} -n ${CLUSTER_NAMESPACE} -o yaml &> "${ARTIFACT_DIR}/hive_install_job.yaml" || true
```
Comment (staebler) [inline hack/e2e-test.sh:228]: Remove "job".

**Comment by dgoodwin** [comment]
Updated.

**Pull request merged**